src/workerd/jsg/util.c++

```diff
@@ -674,6 +674,16 @@ kj::Array<kj::byte> asBytes(v8::Local<v8::ArrayBufferView> arrayBufferView) {
   }
 }
 
+kj::Array<kj::byte> asBytes(v8::Local<v8::SharedArrayBuffer> sharedArrayBuffer) {
```

Comment:
Nit: This likely needs a todo comment about adding memory protection key coverage later. Specifically, if the kj::Array<kj::byte> is used outside of the isolate lock, once we have the MPK enforcement fully in place we'll need to make sure it all Just Works. /cc @erikcorry as an FYI